packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -224,13 +258,29 @@ export const ActivityLibrary = (props: Props) => {
     onQueryChange,
     resetQuery
   } = useSearchFilter(templates, getTemplateDocumentValue)
-  const [debouncedSearchQuery] = useDebounce(searchQuery, 500)
```

Comment:
@tianrunhe I changed the debounced query logic. Because this is used for analytics, let me know if that's ok.